ci/test_safe_docker_run.py

```diff
@@ -408,7 +408,7 @@ def wait_for_container(name: str) -> bool:
         # to the atexit
         for sig in [None, signal.SIGTERM, signal.SIGINT]:
             # Execute the safe docker run script in a different process
-            proc = subprocess.Popen(['./safe_docker_run.py', "--name", container_name, "ubuntu:18.04", "sleep 10"])
+            proc = subprocess.Popen(['python3 safe_docker_run.py', "--name", container_name, "ubuntu:18.04", "sleep 10"])
```

Comment:
May need to separate python3 and the script name into two elements of the list?

Reply:
fixed, thanks!